layouts/index.html

```diff
@@ -12,12 +12,16 @@
         <header>
           <h1 class="usa-display">{{ $translation.homepage.header }}</h1>
         </header>
+      {{ if eq $.Site.Params.language "english" }}
         <form class="form-register" action="/states/" method="get">
+      {{ else }}
+        <form class="form-register" action="/es/states/" method="get">
+      {{ end }}
           <p>{{ $translation.homepage.state_selection.label }}</p>
           <label class="usa-sr-only" for="js-user-selection">
             {{ $translation.homepage.state_selection.default }}
           </label>
-          <select name="user-selection" id="js-user-selection">
+          <select name="userselection" id="js-user-selection">
```

Comment:
Yep - this was needed because the dash in `user-selection` broke the rewrite rule.